src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Resources/Icons/CodeInformationPrivate_16x.xaml

```diff
@@ -1,18 +1,14 @@
-﻿<!-- This file was generated by the AiToXaml tool.-->
-<!-- Tool Version: 14.0.22307.0 -->
-<Viewbox Width="16" Height="16" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+﻿<Viewbox Width="16" Height="16" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
   <Rectangle Width="16" Height="16">
     <Rectangle.Fill>
       <DrawingBrush>
         <DrawingBrush.Drawing>
           <DrawingGroup>
-            <DrawingGroup.Children>
-              <GeometryDrawing Brush="#00FFFFFF" Geometry="F1M16,16L0,16 0,0 16,0z" />
-              <GeometryDrawing Brush="#FFF6F6F6" Geometry="F1M16,12.0068L16,15.9998 9,15.9998 9.005,12.0158C9.005,12.0158,8.983,11.5798,9.087,10.9998L8.949,10.9998C8.968,11.1658 9,11.3288 9,11.4998 9,13.9818 6.981,15.9998 4.5,15.9998 4.47,15.9998 4.442,15.9918 4.412,15.9908 4.382,15.9918 4.354,15.9998 4.325,15.9998 1.94,15.9998 0,14.0598 0,11.6748 0,11.6418 0.00699999999999967,11.6108 0.00799999999999912,11.5778 0.00699999999999967,11.5518 0,11.5268 0,11.4998 0,9.5468 1.258,7.8978 3,7.2768L3,4.9998 1,4.9998 1,0.9998 7,0.9998 8,0.9998 15,0.9998 15,3.9998 15,4.9998 15,7.9998 11,7.9998 11,8.3388C11.434,8.1508 11.952,8.0308 12.594,8.0308 14.126,8.0308 16,8.9978 16,12.0068" />
-              <GeometryDrawing Brush="#FF424242" Geometry="F1M13,14L12,14 12,13 13,13z M14,12L11,12 10,12 10,15 15,15 15,12z M10.018,11.563C10.148,10.438 10.198,9.031 12.594,9.031 14.82,9.031 14.879,10.443 14.964,11.563L13.998,11.563C13.984,10.816 13.843,10.021 12.594,10.021 11.353,10.021 11.078,10.833 11.019,11.563z M7,4L2,4 2,2 7,2z M9.381,10L8.724,10C8.439,9.204,7.941,8.519,7.293,8L10,8 10,9.039C9.728,9.334,9.524,9.665,9.381,10 M9,7L4,7 4,5 9,5z M10,5L14,5 14,7 10,7z M14,4L8,4 8,2 14,2z M5,10L4,10 4,9 5,9z M5,14L4,14 4,11 5,11z M4.5,8C2.567,8 1,9.567 1,11.5 1,13.433 2.567,15 4.5,15 6.433,15 8,13.433 8,11.5 8,9.567 6.433,8 4.5,8" />
-              <GeometryDrawing Brush="#FFF0EFF1" Geometry="F1M4,10L5.001,10 5.001,9 4,9z M4,14L5.001,14 5.001,10.999 4,10.999z" />
-              <GeometryDrawing Brush="#FFEFEFF0" Geometry="F1M12,14L13,14 13,13 12,13z" />
-            </DrawingGroup.Children>
+            <GeometryDrawing Brush="#00FFFFFF" Geometry="F1M16,16L0,16 0,0 16,0z" />
```

Comment:
Assuming this is just removing "Children" designation?

Reply:
Yep. It's the XAML content property, so defaulted. These XAML files are copied verbatim into the assemblies.